packages/calypso-build/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Breaking: drop support for wepback 4
 - Breaking: renamed option `output-jsonp-function` to `output-chunk-loading-global`
+- Breaking `./webpack/file-loader` loader doesn't support options anymore. It uses the new `asset/resource` instead of `file-loader`
```

Comment:
```suggestion
- Breaking: `./webpack/file-loader` loader doesn't support options anymore. It uses the new `asset/resource` instead of `file-loader`
```